_docs/releasing_version.md

```diff
@@ -6,69 +6,100 @@ summary: How to perform a complete version release, including modulesync and pub
 ---
 
 Creating a release is a two step process:
-1. Prepare the release — setup everything so that peer review can happen and when everything is ready…
+1. Prepare the release
 2. Do the actual release
 
 ## Preparing a release
 
-Run modulesync to ensure the dotfiles are up-to-date.
+Before preparing a release, it may help to run modulesync to ensure the
+dotfiles are up-to-date. If modulesync has been run recently by someone else,
+you may not have to do it. See https://github.com/voxpupuli/modulesync for more
+info on using modulesync.
```

Comment:
do you think this is fine or should we enhance it with a sentence so people can check if it's actually up2date? each of our modules has the `.msync.yml` file with the version:
```
$ cat .msync.yml
---
# Managed by modulesync - DO NOT EDIT
# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/

modulesync_config_version: '5.5.0'
```
Which maps to the tag at: https://github.com/voxpupuli/modulesync_config/tags

also maybe it's better to link to https://github.com/voxpupuli/modulesync_config/ and not https://github.com/voxpupuli/modulesync ?